tgui/packages/tgui/interfaces/common/InputButtons.tsx

```diff
@@ -0,0 +1,83 @@
+import { useBackend } from '../../backend';
+import { Box, Button, Stack } from '../../components';
+
+type InputButtonsData = {
+  preferences: Preferences;
+};
+
+type InputButtonsProps = {
+  input: string | number | null;
+  inputIsValid: Validator;
+};
+
+export type Validator = {
+  isValid: boolean;
+  error: string | null;
+};
+
+export type Preferences = {
+  large_buttons: boolean;
+  swapped_buttons: boolean;
+};
+
+/** The buttons shown at bottom. Will display the error
+ * if the input is invalid.
+ */
```

Comment:
Same, this is not documentation, and don't write on the top part `/**` if you do.